support_bundle/get_support_bundle.sh

```diff
@@ -195,6 +195,9 @@ main() {
     echo "Using namespace ${NAMESPACE}"
     echo "Using context ${CONTEXT}"
 
+    # Collect kubectl version
+    kubectl version > ${LOG_DIR}/kubectl_version.txt
```

Comment:
this should also use the context so that we can capture the server version

Reply:
Thanks Mark, should be all set.